app/assets/javascripts/miq_tree.js

```diff
@@ -363,19 +415,17 @@ function miqSquashToggle(treeName) {
 
 function miqTreeEventSafeEval(func) {
```

Comment:
My guess is that this function is not called anywhere anymore, so you can delete it safely if you're not planning to use it.

Reply:
What about the functions in the tree which are calling it? Can I delete them as well? And then the option from the tree?

Reply:
See above my comment :wink: